CHANGELOG_DESKTOP.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [1.42.86](https://github.com/brave/brave-browser/releases/tag/v1.42.86)
+
+ - Added Aurora EVM chain to preloaded chains for Brave Wallet. ([#23522](https://github.com/brave/brave-browser/issues/23522))
+ - Added loading skeletons to Brave Wallet panel. ([#23577](https://github.com/brave/brave-browser/issues/23577))
+ - Added support for using Ledger hardware for Solana signTransaction, signAllTransactions provider APIs with Brave Wallet. ([#23842](https://github.com/brave/brave-browser/issues/23842))
```

Comment:
Probably can update to `Added support for using Ledger hardware for Solana Sign Transactions using Brave wallet`

Reply:
Updated